docs/open-api-spec/openapi_original.yaml

```diff
@@ -0,0 +1,5101 @@
+openapi: 3.0.0
```

Comment:
Do we need to keep the original open API spec? Shall we keep only the modified version as it is the spec for our module.

Reply:
Removed in [e2fe6c8](https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/pull/241/commits/e2fe6c8dbae471bc0ee511bb42eb69a4e891b54b)